test/fixtures/order_items.yml

```
@@ -32,4 +32,10 @@ order_item_five:
   product: ball
   order: order_pending
   quantity: 2
-  shipped: false
\ No newline at end of file
+  shipped: false
+
+order_item_too:
+  product: broom
+  order: complete_order
```

This might break some of Kate's tests around total revenue by status, but it should be simple to fix! If you want to be extra safe, this could be changed to pending_order (Kate isn't using that one in her tests).

```suggestion
  order: pending_order
```